### ISSUE TITLE
Tornado requirement

### ISSUE THREAD

**Issue by paulocheque** [circus-tent/circus#1030]
I am receiving:

```
File "env/lib/python3.4/site-packages/tornado/ioloop.py", line 48, in <module>
ImportError: No module named 'tornado.platform'
```

- CentOS
- python 3.4
- circus==0.14.0
- tornado==4.4.2

Is a compatibility issue with Tornado? the `setup.py` of Circus just say **tornardo >= 3.0**.

Thanks in advance

**Comment by paulocheque** [comment]
Yeah, that's it.

Related:
- https://github.com/circus-tent/circus/pull/817
- https://github.com/circus-tent/circus/issues/865
- https://github.com/circus-tent/circus/issues/872

**Comment by k4nar** [comment]
Should be fixed by the release [0.15.0](https://github.com/circus-tent/circus/releases/tag/0.15.0).
